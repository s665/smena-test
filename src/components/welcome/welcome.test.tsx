import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from "./welcome";

it('render correctly', () => {
    const three = renderer
        .create(<Welcome/>)
        .toJSON()
    expect(three).toMatchSnapshot()
})