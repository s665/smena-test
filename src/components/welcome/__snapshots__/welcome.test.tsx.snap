// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Добро пожаловать!
  </h1>
  <p>
    Для просмотра информации вам необходимо зарегистрироваться или авторизоваться
  </p>
  <div
    className="ant-space ant-space-horizontal ant-space-align-center"
    style={Object {}}
  >
    <div
      className="ant-space-item"
      style={
        Object {
          "marginRight": 8,
        }
      }
    >
      <button
        className="ant-btn ant-btn-primary"
        onClick={[Function]}
        type="button"
      >
        <span>
          Вход
        </span>
      </button>
    </div>
    <div
      className="ant-space-item"
      style={Object {}}
    >
      <button
        className="ant-btn"
        onClick={[Function]}
        type="button"
      >
        <span>
          Зарегистрироваться
        </span>
      </button>
    </div>
  </div>
</div>
`;
